platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -1391,13 +1396,17 @@ private Response continueLogout(
       } else {
         // No more targets, respond to original issuer
         entityId = logoutState.getOriginalIssuer();
-        String status =
-            logoutState.isPartialLogout() ? StatusCode.PARTIAL_LOGOUT : StatusCode.SUCCESS;
         logoutObject =
-            logoutMessage.buildLogoutResponse(
-                SystemBaseUrl.constructUrl(IDP_LOGIN, true),
-                status,
-                logoutState.getOriginalRequestId());
+            logoutState.isPartialLogout()
```

Comment:
✏️ Alternatively you can put the ternary inside of the `buildLogoutResponse()` call:
```
        logoutObject =
            logoutMessage.buildLogoutResponse(
                SystemBaseUrl.constructUrl(IDP_LOGIN, true),
                logoutState.isPartialLogout() ? StatusCode.PARTIAL_LOGOUT : StatusCode.SUCCESS,
                logoutState.getOriginalRequestId());
```